stable/datadog/values.yaml

```diff
@@ -279,6 +279,36 @@ datadog:
 #      cpu: 200m
 #      memory: 256Mi
 
+## @param systemProbe - object - required
+## Enable systemProbe agent and provide custom configs
+#
+systemProbe:
+  ## @param enabled - boolean - required
+  ## Set this to true to enable system-probe agent
+  #
+  enabled: false
+
+  ## @param debugPort - integer - required
+  ## Specify the port to expose pprof and expvar for system-probe agent
+  #
+  debugPort: 0
+
+  ## @param enableConntrack - boolean - required
+  ## Enable the system-probe agent to connect to the netlink/conntrack subsystem to add NAT information to connection data
+  ## Ref: http://conntrack-tools.netfilter.org/
+  #
+  enableConntrack: true
+
+  ## @param seccompRoot - string - required
+  ## Specify the seccomp profile root directory
+  #
+  seccompRoot: /var/lib/kubelet/seccomp
+
+  ## @param bpfDebug - boolean - optional
+  ## Enable logging for kernel debug
+  #
+  # bpfDebug: false
```

Comment:
Shouldn't this be uncommented and turned into a required parameter?
Otherwise `bpf_debug: {{ $.Values.systemProbe.bpfDebug }}` would be rendered as `bpf_debug: `.